### ISSUE TITLE
Media::setNewOrder() goes wrong

### ISSUE THREAD

**Issue by thorem** [spatie/laravel-medialibrary#106]
Media::setNewOrder($ids, 1) returns the next error:
```
"ErrorException in MimeType.php line 26: finfo::buffer() expects parameter 1 to be string, resource given"
```

I think it goes wrong on the save() action, because when i manually updating a media record i get the same error: 
```php
$media->update(['order_column' => 1]);
```

**Comment by freekmurze** [comment]
This should be fixed in v3.10.2. If you still are having problems while using that version, reopen this issue.

**Comment by thorem** [comment]
Thank you for the quick fix
